slashCommands/vote.ts

```diff
@@ -145,14 +163,26 @@ async function printVotes(pro: Set<string>, con: Set<string>, reply: Message, ti
         msgEmbed.setDescription(title);
     }
     //show embed
-    await reply.edit({embeds:[msgEmbed]})
+    await reply.edit({embeds:[msgEmbed], components:[]})
+}
+
+function playercanstartvote (interaction: CommandInteraction, messagerolepos: number): boolean {
+    const roles = interaction.member.roles as GuildMemberRoleManager;
+    let phrp = 0 //player highest role position
+    roles.cache.forEach((r)=> {
+        if (r.position > phrp) {
+            phrp = r.position;
+        }
+    });
+    return messagerolepos > phrp
 }
 
-function getEmbedOptions(title:string, msg: string, timestamp?:number): BaseMessageOptions {
+function getEmbedOptions(title:string, msg: string, group:number, timestamp?:number): BaseMessageOptions {
```

Comment:
As far as I'm aware, because of the large size of id's in discord, it might be too large for a regular integer. While I dont know how js handles that, I think we should just use discord.js's `Snowflake` (which is just their fancy word for their IDs) for things like this

Reply:
javascript only uses 53 bit. python wouldnt have this problem ^^